src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/products.html.twig

```diff
@@ -57,82 +64,168 @@
+              {{ 'Cancel'|trans({}, 'Admin.Actions') }}
+            </button>
+            <button type="button" class="btn btn-sm btn-primary js-product-add-action-btn mt-2 mb-2 d-inline" id="addProductActionBtn" data-order-id="{{ orderForViewing.id }}" disabled>
+              {{ 'Add'|trans({}, 'Admin.Actions') }}
+            </button>
+          </td>
+        </tr>
+        <tr id="editProductTableRow" class="d-none">
+          <td></td>
+          <td></td>
+          <td class="pr-5">
+            <input type="hidden" class="editOrderDetailId">
+            <input type="hidden" class="editProductTaxRate">
+            <div class="input-group">
+              <input type="text" class="form-control editProductPriceTaxExcl">
+              <div class="input-group-append">
+                <div class="input-group-text">{{ orderCurrency.symbol }} {{ 'tax excl.'|trans({}, 'Admin.Global') }}</div>
+              </div>
+            </div>
+            <div class="input-group mt-2">
+              <input type="text" class="form-control editProductPriceTaxIncl">
+              <div class="input-group-append">
+                <div class="input-group-text">{{ orderCurrency.symbol }} {{ 'tax incl.'|trans({}, 'Admin.Global') }}</div>
+              </div>
+            </div>
+          </td>
+          <td class="pr-5"><input type="number" class="editProductQuantity form-control" min="1" value="1"></td>
+          <td class="editProductAvailable"></td>
+          <td colspan="3">
+            <button type="button" class="btn btn-sm btn-secondary js-product-edit-action-btn mr-2 d-inline cancelEditProductBtn">
+              {{ 'Cancel'|trans({}, 'Admin.Actions') }}
+            </button>
+            <button type="button" class="btn btn-sm btn-primary js-product-edit-action-btn mt-2 mb-2 d-inline editProductActionBtn" data-order-id="{{ orderForViewing.id }}">
+              {{ 'Save'|trans({}, 'Admin.Actions') }}
+            </button>
+          </td>
+        </tr>
+        <tr id="templateProductTableRow" class="d-none">
+          <td>
+              <img alt="">
+          </td>
+          <td>
+            <a href="">
+              <p class="mb-0"></p>
+              <p class="mb-0">
+                {{ 'Reference number:'|trans({}, 'Admin.Orderscustomers.Feature') }}
+              </p>
+              <p class="mb-0">
+               {{ 'Supplier reference:'|trans({}, 'Admin.Orderscustomers.Feature') }}
+              </p>
+            </a>
+          </td>
+          <td></td>
+          <td></td>
+          <td>
+            <span class="badge badge-secondary rounded-circle"></span>
+            <div class="d-none js-product-quantity">
+              <input type="text" value="" class="form-control">
+            </div>
+          </td>
+          <td></td>
+          <td></td>
+            <td class="text-right">
+              <a href="#"
```

Comment:
If this link is supposed to be managed by javascript it should not contain an `href='#'` And the best (the crem of the crem) would be to use use a `<button type="button">` It avoids using `return false` or `event.preventDefault` stuffs in the js callback (no scroll position moved, and no form submit!)

Reply:
I replaced by a `href="javascript:void(0);"`